capa/features/extractors/vmray/call.py

```diff
@@ -30,40 +35,78 @@ def extract_call_features(ph: ProcessHandle, th: ThreadHandle, ch: CallHandle) -
 
     # TODO (meh): update for new models https://github.com/mandiant/capa/issues/2148
     # print(ch)
-    return
 
     # Extract API name
-    yield API(ch.inner.name), ch.inner.address
-
+    yield API(FunctionCall.fncall_id), Address(FunctionCall.addr)
+    
+    # Map str "type" values as int if they are in list
+    datatype = [
```

Comment:
ideally we move this to the model and support Literals there (and do pre/postprocessing to convert to numbers where applicable)